bundler/lib/bundler/cli.rb

```diff
@@ -366,6 +366,7 @@ def binstubs(*gems)
     method_option "version", :aliases => "-v", :type => :string
     method_option "group", :aliases => "-g", :type => :string
     method_option "source", :aliases => "-s", :type => :string
+    method_option "require", :aliases => "-r", :type => :string
```

Comment:
Would it be possible to add banner with info about possible values (path or `false`)?